### PR TITLE
Added newsletter button to home

### PR DESCRIPTION
**Pull request by caioertai**
Added the button to the home. Not sure about the color tho...

### CODE REVIEW

**Comment by andrerferrer** [comment]
Really good code!

**Comment by caioertai** [comment]
I think the color should be primary...

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1 +1,3 @@
 <h1>Hello World</h1>
+
+<%= link_to  "Subscribe to our Newsletter", "#", class: "btn btn-warning" %>
```
Comment (andrerferrer) [inline app/views/pages/home.html.erb]: This is sexier if we change the class to btn-primary

**Review by andrerferrer — CHANGES_REQUESTED** [review]

File: app/views/pages/home.html.erb [hunk 2]
```diff
@@ -1 +1,3 @@
 <h1>Hello World</h1>
+
```
Comment (caioertai) [inline app/views/pages/home.html.erb:2]: Ugly extra line....

File: app/views/pages/home.html.erb (hunk 1 above)
Comment (caioertai) [inline app/views/pages/home.html.erb]: I think it should be primary...

**Review by andrerferrer — APPROVED** [review]

**Pull request merged**